contrib/cirrus/README.md

```diff
@@ -0,0 +1,286 @@
+![PODMAN logo](../../logo/podman-logo-source.svg)
+
+# Cirrus-CI
+
+Similar to other integrated github CI/CD services, Cirrus utilizes a simple
+YAML-based configuration/description file: ``.cirrus.yml``.  Ref: https://cirrus-ci.org/
+
+
+## Workflow
+
+All tasks execute in parallel, unless there are conditions or dependencies
+which alter this behavior.  Within each task, each script executes in sequence,
+so long as any previous script exited successfully.  The overall state of each
```

Comment:
```suggestion
which alter this behavior. Within each task, each script executes in sequence,
so long as any previous script exited successfully. The overall state of each
```

double spaces

Reply:
Ooops this is not supposed to be in this PR.